doc/user/content/sql/tail.md

```diff
@@ -42,13 +42,14 @@ Field | Represents
 
 ### AS OF
 
-You can choose a point in time to begin reporting events using `AS OF`.
-If you don't use `AS OF`, Materialize will pick a timestamp itself.
+`AS OF` is the specific point in time to start reporting all events for a given `TAIL`. If you don't
+use `AS OF`, Materialize will pick a timestamp itself.
 
-### WITH SNAPSHOT
+### WITH SNAPSHOT or WITHOUT SNAPSHOT
 
-If you specify `WITH SNAPSHOT`, a snapshot of the state of the system at the chosen timestamp will be reported in addition to any subsequent events.
-This means that you will see any updates that occur up to and including the chosen timestamp.
+By default, each `TAIL` is created with a `SNAPSHOT`: a compacted version of the updates before its
+`AS OF` timestamp. After the `SNAPSHOT`, all uncompacted updates will be sent to the `TAIL`. To only see
```

Comment:
q re: `all uncompacted updates`: Does this mean all updates will be sent to the `TAIL` in an uncompacted form? Or that if an update is not compacted, it will be sent to the `TAIL`, but if it _is_ compacted, it will not be? If the former, I would suggest a slight rewording.

Reply:
If `WITH SNAPSHOT`, `TAIL` will receive compacted updates up until the `AS OF` timestamp and all uncompacted updates after. If `WITHOUT SNAPSHOT`, `TAIL` will not start up with compacted updates and will only see data after the `AS OF` timestamp.

Reply:
My question is more around what are uncompacted updates? Are all updates uncompacted? Are some updates compacted, and you won't see them in `TAIL` after you start it using `WITH SNAPSHOT`?

Reply:
Uncompacted updates are just normal diffs that would be sent to a `TAIL` -- all normal updates are uncompacted. We only have compacted updates when trying to catch up to a particular time (so showing the cumulative diff for each value, instead of streaming each diff).

Reply:
Maybe (guessing) Sean's point is that no one except us knows what "compacted" and "uncompacted" mean. What about something like

> By default, each `TAIL` is created with a `SNAPSHOT` which contains the results of the query at its `AS OF` timestamp. Any further updates to these results are produced at the time when they occur.